src/documents/showcase/Submarine Duel.md

```diff
@@ -0,0 +1,7 @@
+```
+layout: showcase
+title: "Submarine Duel"
+android: https://play.google.com/store/apps/details?id=com.uglypixels.submarineduel
+ios: https://itunes.apple.com/us/app/submarine-duel/id1081035768
+amazon: https://www.amazon.com/dp/B01B5DKLOA/
```

Comment:
Can you remove this? It won't work, there's no check for `amazon` anywhere. There should also be a `website` - you can link to either Android or iOS if you don't have a dedicated website for the game, whichever you prefer (it's where you're linked to when you click the image).